lib/household/view/view_respondent.dart

```diff
@@ -115,6 +165,37 @@ Widget respondentView(BuildContext context, Respondent respondent) {
   );
 }
 
+class AnthropometricsOptions extends StatelessWidget {
```

Comment:
can this be put inside the above widget

Reply:
Since this is constructing a bottom sheet rather than a widget inside the existing scaffold I'd prefer to keep it separate